api/org/commcare/api/json/Walker.java

```diff
@@ -0,0 +1,95 @@
+package org.commcare.api.json;
+
+import org.javarosa.core.model.FormIndex;
+import org.javarosa.form.api.FormEntryController;
+import org.javarosa.form.api.FormEntryModel;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+/**
+ * Created by willpride on 12/8/15.
```

Comment:
What does a `Walker` walk over / do?
I would mention somewhere that it actually builds a `JSONArray` of a form (?) (not even sure what that means)